From 7cf91afe36e807ce27d132d7c9cec877c2c08feb Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 14 Jun 2012 18:07:30 +0800
Subject: ENGR00212720 [MX6]Adjust CPU 672M setpoint voltage

Previous voltage for 672M is 1.05V, normal test is OK,
but if CPU is busy in background and do the CPUFreq change
as well, always fail the stress test at 672M setpoint, after
increase it to 1.1V, stress test is OK.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index 6fe2fd8..faa65fa 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -66,7 +66,7 @@ static struct cpu_op mx6_cpu_op_1G[] = {
 	{
 	 .pll_rate = 672000000,
 	 .cpu_rate = 672000000,
-	 .cpu_voltage = 1050000,},
+	 .cpu_voltage = 1100000,},
 	 {
 	  .pll_rate = 396000000,
 	  .cpu_rate = 396000000,
-- 
1.7.9.5

